scripts/buildTs.mjs

```diff
@@ -125,12 +125,13 @@ try {
   await Promise.all(
     packagesWithTs.map(({packageDir, pkg}) =>
       mutex(async () => {
-        const buildDir = path.resolve(packageDir, 'build/**/*.d.ts');
```

Comment:
Also this should not work on Windows, because the resulting pater will have `\` separators. Hence is should not select any files.

`glob` has the `cwd` option to solve this problem.